tempsensor/sensor_daemon.hpp

```diff
@@ -0,0 +1 @@
+int *nop(void);
```

Comment:
if you're not using `nop` for anything you can drop it, i had it in the other daemon as a placeholder until the thread functions got implemented.

Reply:
I just ended up removing the whole header file in the next commit to fix this.